packages/fast-usdc-contract/src/fast-usdc.contract.ts

```diff
@@ -193,9 +193,33 @@ export const contract = async (
   });
 
   const zoeTools = makeZoeTools(zcf, vowTools);
+  const { advanceFunds } = orchestrateAll(
+    // @ts-expect-error flow membrance type debt
+    { advanceFunds: flows.advanceFunds },
+    {
+      chainHub, // XXX excess authority?
+      feeConfig,
+      getNobleICA,
+      log: makeTracer('AdvanceFunds'),
+      settlementAccount: settleAccountV,
+      statusManager,
+      usdc: harden({
+        brand: terms.brands.USDC,
+        denom: terms.usdcDenom,
+      }),
+      zcfTools: harden({
+        makeEmptyZCFSeat: () => {
+          const { zcfSeat } = zcf.makeEmptySeatKit();
+          return zcfSeat;
+        },
+      }),
+      zoeTools,
```

Comment:
These are similarly bag of functions that need to be wrapped with endowment kinds that appear in baggage. This is particularly concerning for zoeTools which is simply passed through here (any future removal of a tool would prevent future upgrades)

Reply:
Is there a viable alternative?

Future upgrades of an in-flight flow. We should be able to settle outstanding Fast USDC flows should we need to remove something.

Reply:
The only alternative is defining a durable exo containing these as methods.